final_project/01-sales-flow.py

```diff
@@ -0,0 +1,71 @@
+"""
+Sales processing pipeline
+"""
+import datetime as dt
+from datetime import timedelta
+from airflow import DAG
+from airflow.providers.google.cloud.operators.bigquery import BigQueryInsertJobOperator
+from table_defs.sales_bronze_csv import sales_bronze_csv
+
+
+DEFAULT_ARGS = {
+    'depends_on_past': True,
```

Comment:
each day in sales is independent. Also, you don't use MERGE here etc, so no sense to set depends_on_past to True. It slows the processing only